### PR TITLE
feat(terramate-ls): implements terramate.createStack command.

### PR DESCRIPTION
**Pull request by i4ki**
The command supports the arguments below:

- `uri=`                target directory.
- `genid=`              any value, if set, generates an ID.
- `name=`               name of the stack.
- `description=`        description of the stack.

### CODE REVIEW

File: ls/commands.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+			if err != nil {
+				return errors.E(ErrCreateStackFailed, err, "generating ID")
+			}
+			stackConfig.ID = id.String()
+		case "name":
+			stackConfig.Name = argVal
+		case "description":
+			stackConfig.Description = argVal
+		default:
+			panic("unreachable")
```
Comment (wmalik) [inline ls/commands.go:94]: Would this crash the language server? Would it make sense to return an `ErrUnrecognizedAttr` instead?

File: ls/ls.go [hunk 2]
```diff
@@ -129,6 +135,9 @@ func (s *Server) handleInitialize(
 					IncludeText: false,
 				},
 			},
+			// ExecuteCommandProvider: &lsp.ExecuteCommandOptions{
```
Comment (wmalik) [inline ls/ls.go:138]: Is it intentionally commented out?

**Review by wmalik — COMMENTED** [review]
Looks great, some minor comments/questions.

File: ls/commands.go (hunk 1 above)
Comment (i4ki) [reply ls/commands.go:94]: yes, I added the panic while testing and forgot to improve. Thanks!

**Review by wmalik — APPROVED** [review]
Looks good!

**Pull request merged**